app/src/main/java/com/es/findsoccerplayers/MainActivity.java

```diff
@@ -1,14 +1,74 @@
 package com.es.findsoccerplayers;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.widget.Toast;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static final String TAG = "Main Activity";
+    private FirebaseAuth mAuth;
+    private Toolbar toolbar;
+    private long backPressedTime;
+    private Toast backToast;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        Log.w(TAG,"Activity Creata");
+        mAuth=FirebaseAuth.getInstance();
+        FirebaseUser user = mAuth.getCurrentUser();
+        if(user==null) {
```

Comment:
se `user == null` fai partire la loginActivity? La mia idea è che MainActivity parte solo se il login va a buon fine

Reply:
quindi dovrei far partire loginActivity come prima activity? e in caso ci sia un utente loggato parte la main?

Reply:
si io farei cosi